ee/clickhouse/views/person.py

```diff
@@ -116,18 +123,20 @@ def calculate_funnel_correlation_persons(
             return {"result": ([], None, None)}
 
         filter = Filter(request=request, data={"insight": INSIGHT_FUNNELS}, team=self.team)
+        if not filter.correlation_person_limit:
+            filter = filter.with_data({FUNNEL_CORRELATION_PERSON_LIMIT: 100})
         base_uri = request.build_absolute_uri("/")
-        people, should_paginate = FunnelCorrelationPersons(filter=filter, team=self.team, base_uri=base_uri).run()
+        people, _ = FunnelCorrelationPersons(filter=filter, team=self.team, base_uri=base_uri).run()
```

Comment:
ah, this one's correct here, but below should return the serialized_people, not the raw ones

Reply:
This one and the one below isn't using the actor pattern yet. Only the base funnel step queries are. So the `people` variable is the serialized result